src/as-validator.c

```diff
@@ -882,7 +882,7 @@ as_validator_check_description_tag (AsValidator *validator, xmlNode* node, AsFor
  * as_validator_check_appear_once:
  **/
 static void
-as_validator_check_appear_once (AsValidator *validator, xmlNode *node, GHashTable *known_tags)
+as_validator_check_appear_once (AsValidator *validator, xmlNode *node, GHashTable *known_tags, gboolean allowTranslations)
```

Comment:
style: Please use snake_case => `allow_translations` - but for conciseness, I'd probably name this `translatable` or `allow_l10n` (your choice though) :)